### PR TITLE
fix nbody indexing error & add unit test

### PR DESCRIPTION
**Pull request by sblunt**
Fix for issue #357

### CODE REVIEW

**Comment by sblunt** [comment]
The nbody code was throwing an indexing error when used in an MCMC run (issue #357). I fixed the error and added a unit test. I also made the choice of nbody backend integrator a keyword.

File: orbitize/nbody.py [hunk 1]
```diff
@@ -16,104 +28,115 @@ def calc_orbit(
         inc (np.array): inclination [radians]
         aop (np.array): argument of periastron [radians]
         pan (np.array): longitude of the ascending node [radians]
-        tau (np.array): epoch of periastron passage in fraction of orbital period 
+        tau (np.array): epoch of periastron passage in fraction of orbital period
             past MJD=0 [0,1]
         plx (np.array): parallax [mas]
-        mtot (np.array): total mass of the two-body orbit (M_* + M_planet) 
+        mtot (np.array): total mass of the two-body orbit (M_* + M_planet)
```
Comment (semaphoreP) [inline orbitize/nbody.py:34]: should `mtot` be an float, instead of an array?

File: orbitize/nbody.py [hunk 2]
```diff
@@ -16,104 +28,115 @@ def calc_orbit(
...
         m_pl (np.array, optional): mass of the planets[units]
         output_star (bool): if True, also return the position of the star
             relative to the barycenter.
+        integrator (str): value to set for rebound.sim.integrator. Default "ias15"
 
     Returns:
         3-tuple:
 
-            raoff (np.array): array-like (n_dates x n_orbs) of RA offsets between 
+            raoff (np.array): array-like (n_dates x n_bodies x n_orbs) of RA offsets between
```
Comment (semaphoreP) [inline orbitize/nbody.py:46]: is n_orbs always 1? if so, we should specific that.

**Review by semaphoreP — APPROVED** [review]
Unit test looks good. I just have a couple questions on the docstring

File: orbitize/nbody.py (hunk 2 above)
Comment (sblunt) [reply orbitize/nbody.py:46]: yes, good catch

File: orbitize/nbody.py (hunk 1 above)
Comment (sblunt) [reply orbitize/nbody.py:34]: yes, good catch. Thanks!

**Comment by sblunt** [comment]
Thanks for catching the docstring errors. I updated it to be consistent with the way the function works.

**Pull request merged**